src/appmixer/freshdesk/tickets/DeletedTicket/DeletedTicket.js

```
@@ -0,0 +1,62 @@
+'use strict';
+const axios = require('axios');
+
+module.exports = {
+    async tick(context) {
+        const { auth } = context;
+        
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)